extensions/resource-deployment/src/ui/deployClusterWizard/pages/serviceSettingsPage.ts

```diff
@@ -175,46 +175,58 @@ export class ServiceSettingsPage extends ResourceTypePage {
 		this.endpointHeaderRow = createFlexContainer(view, [this.endpointNameColumnHeader, this.dnsColumnHeader, this.portColumnHeader]);
 
 		this.controllerNameLabel = createLabel(view, { text: localize('deployCluster.ControllerText', "Controller"), width: labelWidth, required: true });
-		this.controllerDNSInput = createInputBox(view, { ariaLabel: localize('deployCluster.ControllerDNSName', "Controller DNS name"), required: false, width: inputWidth });
-		this.controllerPortInput = createNumberInput(view, { ariaLabel: localize('deployCluster.ControllerPortName', "Controller port"), required: true, width: NumberInputWidth, min: 1 });
+		const controllerDNSInput = createInputBox(view, { ariaLabel: localize('deployCluster.ControllerDNSName', "Controller DNS name"), required: false, width: inputWidth });
```

Comment:
Are these changes which seem to introducing a constant necessary?

Reply:
Sort of. createInputBox returns an InputComponentInfo now instead of the component directly. So I could have also updated the controllerPortInput to store the whole object - but given that we don't actually need that I just left it as the component.